api/src/api/routes/v1/brla.route.ts

```diff
@@ -25,6 +24,7 @@ router.route('/triggerOfframp').post(validateBrlaTriggerOfframpInput, brlaContro
 
 router.route('/createSubaccount').post(validataSubaccountCreation, brlaController.createSubaccount);
 
-router.route('/triggerPayIn').post(validateTriggerPayIn, brlaController.triggerPayIn);
+router.route('/startKYC2').post(brlaController.startKYC2);
```

Comment:
Just for the sake of it, why don't we add a `validate...` function like we do for any other post request.